### PR TITLE
feat(mailchimp): add mailchimp integration

### PR DESCRIPTION
**Pull request by vitoUwu**
## What is this contribution about?
This contribution adds a new integration called mailchimp, this integration provides email and marketing automation for growing business 

## Link
https://deco-sites-mailchimp.deno.dev/
https://github.com/deco-sites/mailchimp/

### CODE REVIEW

**Comment by guitavano** [comment]
Little comments, very nice job!!

**Comment by guitavano** [comment]
Any news about use generic fetchSafe?

File: mailchimp/mod.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+   * @description e.g: us1, us18, us21, ...etc
+   */
+  serverPrefix: string;
+}
+/**
+ * @title Mailchimp
+ */
+export default function Mailchimp(props: Props) {
+  const { serverPrefix, apiKey } = props;
+  const key = typeof apiKey === "string" ? apiKey : apiKey.get();
```
[the remark this answers is not in the document]
Comment (vitoUwu) [reply mailchimp/mod.ts]: done

File: decohub/apps/mailchimp.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export { default } from "../../mailchimp/mod.ts";
+import { Markdown } from "../components/Markdown.tsx";
+
+export const Preview = await Markdown(
+  new URL("../../mailchimp/README.md", import.meta.url).href,
+);
```
[the remark this answers is not in the document]
Comment (vitoUwu) [reply decohub/apps/mailchimp.ts]: done

**Pull request merged**